packages/backend-function/src/function_client_config_generator.ts

```diff
@@ -0,0 +1,128 @@
+import fs from 'fs';
+import { EOL } from 'os';
+import path from 'path';
+
+const lambdaConfigHelper = (
+  functionName: string
+) => `import { env } from "../env/${functionName}";
+import modelIntrospection from "../model-introspection-schema/${functionName}.json";
+
+import {
+  CredentialsAndIdentityIdProvider,
+  CredentialsAndIdentityId,
+} from "aws-amplify/auth";
```

Comment:
if we're injecting this into customer's project we should scope as much as we can to avoid collisions with customer symbols.

for example https://github.com/aws-amplify/amplify-backend/blob/10ef35d2fe00025fefd12603bfb2e3dce696be9a/packages/backend-function/src/lambda-shims/resolve_ssm_params.ts#L12 .


Additionally, it would be great to extract content of this snippet to normal `ts` file and parametrize `functionNAme`.

For example:

```
internalInitializeAmplifyDataClient = async (functionName: string) => {
  await import('../env/${functionName}');
  ... rest of this code.
}

```

this will both scope it and get this under compiler checks. and. make it testable perhaps.